src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/PhpConfigReferenceDumpPassTest.php

```diff
@@ -35,6 +35,11 @@ protected function setUp(): void
     protected function tearDown(): void
     {
         if (is_dir($this->tempDir)) {
+            // Make the directory is writable on Windows
+            if ('\\' === DIRECTORY_SEPARATOR && !is_writable($this->tempDir)) {
```

Comment:
@GromNaN as far as I can see this would have worked if the added lines had operated on the `readonly` subdirectory